### ISSUE TITLE
Add Roslyn code style analyzers

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#94]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To have an automated way of keeping the code clean and readable.

### Why

As the solution gets bigger, it requires lots of manual tidying.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
